lib/awful/widget/watch.lua

```diff
@@ -0,0 +1,84 @@
+---------------------------------------------------------------------------
+--- Watch widget.
```

Comment:
i like the idea, to have an example code (fo default rc.lua) and screenshot of the widget attached